src/widget/textbox.rs

```diff
@@ -193,11 +195,11 @@ impl TextBox {
 
     // TODO: waiting on druid clipboard support for copy / paste.
     fn copy_text(&self, input: String) {
-        eprintln!("COPY: {}", input);
+        error!("COPY: {}", input);
```

Comment:
I'd go with `warn!`, and change the message to indicate this is not implemented. 👍